docs/mobile/state-management.md

```diff
@@ -1,7 +1,106 @@
 ## Mobile Playbook: State management
 
-#### Persist Redux/Flux store to disk to survive shutdowns and tombstoning
+State management strategies in React Native are similar to regular React applications: Flux architecture with [Redux](http://redux.js.org/) is a sensible default choice. Alternatives like [MobX](https://github.com/mobxjs/mobx) and [Relay](https://facebook.github.io/relay/) are also interesting, but not discussed at length here.
+
+#### Persist Redux/Flux store to disk
+
+Most mobile applications will need to persist their data stores to achieve a snappy user experience and offline resilience.
+
+The simplest way to achieve persistent state and offline caching is to serialize the store object to JSON and write it to disk via the [AsyncStorage](https://facebook.github.io/react-native/docs/asyncstorage.html) API, and reset the store state from the serialized representation on app startup.
+
+You can use the [redux-storage](https://github.com/michaelcontento/redux-storage) middleware with the [redux-storage-engine-reactNativeAsyncStorage](https://github.com/michaelcontento/redux-storage-engine-reactNativeAsyncStorage) backend to achieve this. For performance reasons, [debouncing](https://github.com/michaelcontento/redux-storage-decorator-debounce) the serialization is usually a good idea.
+
+#### Transient state
+
+Some store keys should not be restored on startup. A good example are "isLoading"-style properties that control loading indicator visibility. If an application crashes while a request is in flight, the app would still display the loading indicator on next launch.
+
+Because state is serialized much more often than it is read, prefer to keep writes simple by writing everything to disk, and remove the unwanted properties when restoring the state.
+
+Some attempted strategies for marking which parts of the store are transient:
+ * Blacklist keys (not recommended, increases maintenance burden)
+ * Have a separate store subtree for transient state and [prune](#pruning) on startup (not recommended, leads to fragmented source-of-truth in reducers, though your mileage may vary)
+ * Prefix transient properties with `_` (or other convention) and use a [JSON.parse reviver](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#Using_the_reviver_parameter) to drop keys that match the convention:
+
+   ```
+   JSON.parse(json, (key, value) => key.startsWith('_') ? undefined : value)
```

Comment:
How is this used in conjunction with `redux-storage-engine-reactNativeAsyncStorage`? It looks like the engine already parses the JSON and doesn't accept any parameters to pass as a reviver parameter.